dropwizard-migrations/src/test/java/io/dropwizard/migrations/DbClearChecksumsCommandTest.java

```diff
@@ -31,23 +31,23 @@ void testRun() throws Exception {
     void testHelpPage() throws Exception {
         final ByteArrayOutputStream out = new ByteArrayOutputStream();
         MigrationTestSupport.createSubparser(clearChecksums).printHelp(new PrintWriter(new OutputStreamWriter(out, UTF_8), true));
-        assertThat(out.toString(UTF_8.name())).isEqualTo(String.format(
-            "usage: db clear-checksums [-h] [--migrations MIGRATIONS-FILE]%n" +
-                "          [--catalog CATALOG] [--schema SCHEMA] [file]%n" +
-                "%n" +
-                "Removes all saved checksums from the database log%n" +
-                "%n" +
-                "positional arguments:%n" +
-                "  file                   application configuration file%n" +
-                "%n" +
-                "named arguments:%n" +
-                "  -h, --help             show this help message and exit%n" +
-                "  --migrations MIGRATIONS-FILE%n" +
-                "                         the file containing  the  Liquibase migrations for%n" +
-                "                         the application%n" +
-                "  --catalog CATALOG      Specify  the   database   catalog   (use  database%n" +
-                "                         default if omitted)%n" +
-                "  --schema SCHEMA        Specify the database schema  (use database default%n" +
-                "                         if omitted)%n"));
+        assertThat(out.toString(UTF_8.name())).isEqualTo(
```

Comment:
Looks like there is a NormalizingNewlines missing

Reply:
Right you are. I think I was flipping between `hasToString()` and this and ended up broken in the middle somewhere!